cli/unregister/unregister.go

```diff
@@ -288,3 +302,65 @@ func RemoveServiceContainers() error {
 		return fmt.Errorf(err_string)
 	}
 }
+
+// backupEventLogs loads eventlogs from anax.db, marshals them into the JSON format
+// and saves the bkp file into horizon folder with name of backup time
+func backupEventLogs() error {
```

Comment:
Please call anax /eventlog REST api to get the eventlog and save the output to a text file.  Check this function to see how it is done. Check the case where all=true, detail=true, selections={}, tailing=false.
https://github.com/open-horizon/anax/blob/master/cli/eventlog/eventlog.go#L66.